From afb9598d1199d91f2686d19e9f4e363bb7fb4e29 Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@st.com>
Date: Thu, 2 Nov 2023 14:12:56 +0100
Subject: [PATCH 002/954] feat(memmap): add RELA section display

When PIE is enabled, it can be handy to display the .rela.dyn section.
Add RELA in the list of symbols parsed, to display this section in
memory map.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git a2fea2e65702bc2c8ce98661d1e085b852f7ddd9 feat(memmap): add RELA section display]
Signed-off-by: Yann Gautier <yann.gautier@st.com>
Change-Id: Ic40da54ce6fb6e281fb1a6627e9b8830993cfb2c
---
 tools/memory/memory/memmap.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/memory/memory/memmap.py b/tools/memory/memory/memmap.py
index 99149b5470..74653cc02f 100755
--- a/tools/memory/memory/memmap.py
+++ b/tools/memory/memory/memmap.py
@@ -99,7 +99,7 @@ def main(
 
     if symbols:
         expr = (
-            r"(.*)(TEXT|BSS|RODATA|STACKS|_OPS|PMF|XLAT|GOT|FCONF"
+            r"(.*)(TEXT|BSS|RODATA|STACKS|_OPS|PMF|XLAT|GOT|FCONF|RELA"
             r"|R.M)(.*)(START|UNALIGNED|END)__$"
         )
         printer.print_symbol_table(
-- 
2.53.0

